okhttp/src/test/resources/okhttp3/osgi/resolveTest.bndrun

```diff
@@ -0,0 +1,13 @@
+-runfw: org.eclipse.osgi
```

Comment:
it feels weird to split responsibility between code and text like this. I'm not sure it is bad per se, but I think sometimes of testcontainers and enjoy seeing the setup logic in one markup (even if that is compiled :) )

Reply:
I have moved the bnd instructions from the bndrun, cnf/build.bnd into code so everything should be in one place now.